TDBSCAN/tdbscan.py

```diff
@@ -131,19 +188,25 @@ def main_run(inner_list_coordinates, Eps, CEps, minPoints):
     for key, value in dict_cluster.items():
         coordinates = []
         tuple_array = []
-        for index, each in enumerate(value):
+        for each in value:
             coordinates.append(list_coordinates[each][1])
-            # format will be in index, (lat, lon)
             tuple_array.append(list_coordinates[each])
         if moveability(coordinates) > move_ability:
             key_exceed.append(key)
         else:
             move_dict_cluster[key] = tuple_array
 
     for key, value in move_dict_cluster.items():
-        dict_cluster, dummy_label = main_run(
-            value, stop_Eps, stop_CEps, stop_minPoints)
-        if len(dict_cluster) > 0:
-            main_label[check_for_key_with_highest_count(dict_cluster)] = 1
+        dict_cluster = main_run(
+            value, stop_eps, stop_ceps, stop_min_points)
+        if dict_cluster:
+            main_label[dict_cluster[max(
+                dict_cluster, key=lambda k: len(dict_cluster[k]))][0]] = 1
```

Comment:
Eh while I did suggest to use the `max(..., key=...)` trick, I think it's still good to wrap that under a function name, since it's easier to one glance and understand in general what this part is trying to do without looking straight into the implementation.

Reply:
got it let me change this to a function